scripts/curio-apt/DEBIAN/postinst

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+case "$1" in
+    configure)
+        # Reload systemd after installation or upgrade
+        systemctl daemon-reload
+
+        if [ -z "$2" ]; then
+            # This is an initial installation
+            systemctl enable curio.service
+            if [ ! -f "/etc/curio.env" ]; then
+              echo "CURIO_LAYERS=gui,post\nCURIO_DB_ALL_FIELDS_ARE_OPTIONAL=true\nCURIO_DB_HOST=yugabyte\nCURIO_DB_USER=yugabyte\nCURIO_DB_PASSWORD=yugabyte\nCURIO_DB_PORT=5433\nCURIO_DB_NAME=yugabyte\n" >/etc/curio.env
```

Comment:
We should also add the option to allow change repo-path here. It is used only for journalling but user should still be able to override it.